website/docs/r/vapp_vm.html.markdown

```diff
@@ -236,7 +255,9 @@ example for usage details. **Deprecates**: `network_name`, `ip`, `vapp_network_n
   relayed). It works by querying DHCP leases on Edge Gateway. In general it is quicker than waiting
   until Guest Tools report IP addresses, but is more constrained. However this is the only option if Guest
   Tools are not present on the VM.
-  
+* `os_type` - (Optional; *v2.8+*) Operating System type. Possible values can be found in [Os Types](#os-types). Required when creating empty VM.
+* `hardware_version` - (Optional; *v2.8+*) Virtual Hardware Version (e.g.`vmx-14`, `vmx-13`, `vmx-12`, etc.). Required when creating empty VM.
+* `boot_image` - (Optional; *v2.8+*) Media name to mount as boot image. On update if value changed to empty it will eject mounted media. 
```

Comment:
Need more explanation in the comment about where this media is searched for. You have made catalog_name optional - how do you know in which catalog the media item is?

Also, there seems to be inconsistency. Here you name it `boot_image`, but in test `mediaName` - why the difference?

Reply:
* I query globally, so in any available catalog.
* boot_image is VM context, media is what you mount to cd/rom as boot image.

Reply:
Explanation added note that it's from any catalog.